back-end-side/Controllers/ShowDataController.cs

```diff
@@ -31,11 +31,7 @@ public IActionResult Put([FromBody] Record model, int id)
         {
             // pirma - nustatom kategorija, tada pakeiciam tam tikru indexo value ir tada sortinam
             model.Category = Sorting.CheckInput(model);
-            int index = UploadController.RecordsFromFile.FindIndex(ind => ind.Equals(id));
-            if (index == -1)
```

Comment:
why you deleted if statement?

Reply:
why you wirited if statement?

Reply:
what wirited means?

Reply:
FindIndex returns index if index is found (index = index returns index), and -1 in other cases. Here you were comparing object with int, object and int will never be equal, therefore it returns -1 all the time, so if statement and findIndex are irrelevant